python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -703,6 +749,11 @@ def _find_stars(self, exposure, background, id_generator):
 
         # Measure everything, and use those results to select only stars.
         self.star_measurement.run(sources, exposure)
+        if self.config.do_normalized_calibration:
+            self.star_normalized_calibration_flux.run(
```

Comment:
Just checking on this: what's the point of running it again here, if we used it for the pseudo-ApCorr thing earlier?

Reply:
This is the code that is configured to only *apply* the normalization to the compensated tophats measured in `star_measurement`.  Will add a comment!